PWGHF/TableProducer/pidCreator.cxx

```diff
@@ -26,6 +26,7 @@ using namespace o2;
 using namespace o2::framework;
 
 struct HfPidCreator {
+  /// single-track tables
```

Comment:
If we add the nSigma variables in the candidate creator tables, I would add the combined ones as dynamic columns in the joinable table with PID variables. We would also not need this task anymore once implemented for all the channels, do you agree @vkucera?

Reply:
Agreed. But I think it would be better to define tables per prong index per species which provides the flexibility of arbitrary table combinations for all decay channels. See my comments.